.npmignore

```diff
@@ -0,0 +1,9 @@
+.github
+.env
+.dockerignore
+docker
+docs
+downloads
+images
+tests
+tools
```

Comment:
Why do we need to skip the tools folder? We won't make any changes in the stuff there?

Reply:
tools folder is only for testing the synpress repo itself. it isn't needed when some other packages will use our framework